zinc/src/sbt-test/source-dependencies/compactify/incOptions.properties

```diff
@@ -0,0 +1 @@
+scalac.options = -Xmax-classfile-name 240
```

Comment:
This is necessary to work around an issue with large file paths in Docker AUFS, limit is 240 bytes. It only happens with 2.11.x because the compactify test makes an intensive use of synthesized classes with huge names.